src/pathfora.js

```diff
@@ -1434,53 +1401,75 @@
       var i;
       var j;
 
-      if (utils.hasClass(widget, 'pf-widget-modal')) {
-        widget.querySelector('.pf-widget-content').style.backgroundColor = colors.background;
-      } else {
-        widget.style.backgroundColor = colors.background;
+      if (colors.background) {
+        if (utils.hasClass(widget, 'pf-widget-modal')) {
+          widget.querySelector('.pf-widget-content').style.backgroundColor = colors.background;
+        } else {
+          widget.style.backgroundColor = colors.background;
+        }
       }
 
-      if (fields.length > 0) {
-        j = fields.length;
-        for (i = 0; i < j; i++) {
-          fields[i].style.backgroundColor = colors.fieldBackground;
+      if (colors.fieldBackground) {
+        if (fields.length > 0) {
+          j = fields.length;
+          for (i = 0; i < j; i++) {
+            fields[i].style.backgroundColor = colors.fieldBackground;
+          }
         }
       }
 
       if (contentUnitMeta) {
-        contentUnit.style.backgroundColor = colors.actionBackground;
-        contentUnitMeta.querySelector('h4').style.color = colors.actionText;
-        contentUnitMeta.querySelector('p').style.color = colors.text;
+        if (colors.actionBackground) {
+          contentUnit.style.backgroundColor = colors.actionBackground;
```

Comment:
can we be 100% sure that contentUnit is an actual element / has style and what not? if for some reason `.pf-content-unit` didnt exist this would blow up? ... even tho it should be there as long as our template doesn't get goofed.